src/widget/wslidercomposed.cpp

```diff
@@ -190,8 +190,14 @@ double WSliderComposed::calculateHandleLength() {
                     mode == Paintable::TILE || m_pHandle->height() == 0.0) {
                 return m_pHandle->width();
             } else if (mode == Paintable::STRETCH_ASPECT) {
+                const qreal handleHeight =
+                    static_cast<double>(m_pHandle->height());
```

Comment:
qreal is double or float depending on the target device. Cast to qreal, here and below.